src/boot/bootctl.c

```diff
@@ -68,6 +68,24 @@ static const char *arg_dollar_boot_path(void) {
         return arg_xbootldr_path ?: arg_esp_path;
 }
 
+static void check_efi_path_duplicates(void) {
+        if (check_efi_duplicates(geteuid() != 0) > 1) {
```

Comment:
we usually prefere an "early-exit" coding style over a "deeply nested" coding style.

i.e. reverse this check and return if <= 0.

Reply:
ok